src/test/java/hu/blackbelt/structured/map/proxy/MapProxyTest.java

```diff
@@ -382,6 +382,62 @@ private Map<String, Object> getSimpleProxyMap() {
         return prepared;
     }
 
+    @Test
+    public void testAddAndRemove() {
+        Map<String, Object> prepared = getSimpleProxyMap();
```

Comment:
If you don't use this variable remove it

Reply:
[a9fd813](https://github.com/BlackBeltTechnology/structured-map-proxy/pull/82/commits/a9fd8133c2a418f5629ad0ae4d1019fb16f11970)